docker-compose.yml

```diff
@@ -34,16 +32,19 @@ services:
   webserver:
     build:
       context: .
-      dockerfile: Dockerfile-runnable
-    network_mode: "host"
+      target: with-spark-optional-dag
+      dockerfile: Dockerfile
     restart: always
     depends_on:
       - scheduler
     environment:
-      AIRFLOW__CORE__SQL_ALCHEMY_CONN: postgresql://fixme:fixme@localhost:5999/airflow
+      AIRFLOW__CORE__FERNET_KEY: 8NE6O6RcTJpxcCkuKxOHOExzIJkXkeJKbRie03a69dI=
+      AIRFLOW__CORE__SQL_ALCHEMY_CONN: postgresql://fixme:fixme@postgres:5432/airflow
       AIRFLOW_USER: fixme
       AIRFLOW_EMAIL: fixme@fix.me
       AIRFLOW_PASSWORD: fixme
+    ports:
+      - 127.0.0.1:8080:8080
```

Comment:
I suggest to shift the port forwarding into `docker-compose.override.yml`, so that only local test set-up will allow localhost to access the Web UI.